test/hca/schema/minimal-test.json

```diff
@@ -34,7 +34,9 @@
       "last": "Doe"
     },
     "privacyAgreementAccepted": true,
-    "isSpanishHispanicLatino": false,
+    "view:demographicCategories": {
+      "isSpanishHispanicLatino": false
```

Comment:
Is this required in the minimal test? If we leave it out, it's like the user didn't check any of the boxes.